.gitignore

```diff
@@ -6,10 +6,13 @@
 /pkg/
 /spec/reports/
 /tmp/
+/.vscode/
 
 # rspec failure tracking
 .rspec_status
 
 .env
 .DS_Store
 .idea
+.rubocop.yml
```

Comment:
Why do we need this change ignoring the `.rubocop.yml` file? We definitely want git to keep tracking it.